oak/server/rust/oak_loader/BUILD

```diff
@@ -23,6 +23,9 @@ package(
 rust_binary(
     name = "oak_loader",
     srcs = glob(["src/**/*.rs"]),
+    crate_features = [
+        "oak_debug",
```

Comment:
Will it be possible to enable this feature from command line (i.e. from `./scripts/run_server`) ?

Reply:
No, deliberately &ndash; from the hash of the built binary it should be possible to be convinced that there can be no information leak via logging.